components/SpotifyInterface/include/SpotifyTypedef.h

```diff
@@ -18,7 +18,7 @@ extern "C"
 #include "mdns.h"
 #include "freertos/queue.h"
 #include "esp_psram.h"
-#include "SpotifyWebAppInfo.h"
+#include "SpotifyWebAppInfo_sample.h"
```

Comment:
this is sample, dont fill it with real info

Reply:
we can fix it